### PR TITLE
refactor: Remove `livechat.js` file

### PR DESCRIPTION
**Pull request by KevLehman**

### CODE REVIEW

File: apps/meteor/app/livechat/server/api/v1/webhooks.ts [hunk 1]
```diff
@@ -70,18 +72,18 @@ API.v1.addRoute(
...
-				Livechat.logger.debug({ response });
+				logger.debug({ response });
 				if (request.status === 200) {
 					return API.v1.success();
 				}
 
 				throw new Error('Invalid status code');
 			} catch (error) {
-				Livechat.logger.error(`Error testing webhook: ${error}`);
+				logger.error(`Error testing webhook: ${error}`);
```
Comment (murtaza98) [inline apps/meteor/app/livechat/server/api/v1/webhooks.ts:86]:
```suggestion
				logger.error({
					msg: 'Error testing webhook',
					error,
				});
```

Comment (KevLehman) [reply apps/meteor/app/livechat/server/api/v1/webhooks.ts:86]: Will leave it as a refactor for later 👀

**Review by murtaza98 — APPROVED** [review]

File: apps/meteor/app/livechat/server/methods/getAgentOverviewData.ts [hunk 2]
```diff
@@ -27,7 +26,6 @@ Meteor.methods<ServerMethods>({
 		}
 
 		if (!options.chartOptions?.name) {
-			Livechat.logger.warn('Incorrect analytics options');
```
Comment (lmauromb) [inline apps/meteor/app/livechat/server/methods/getAgentOverviewData.ts:30]: do we still need to log around here?

File: apps/meteor/app/livechat/server/methods/getAnalyticsOverviewData.ts [hunk 3]
```diff
@@ -27,7 +26,6 @@ Meteor.methods<ServerMethods>({
 		}
 
 		if (!options.analyticsOptions?.name) {
-			Livechat.logger.error('Incorrect analytics options');
```
Comment (lmauromb) [inline apps/meteor/app/livechat/server/methods/getAnalyticsOverviewData.ts:30]: same, we dont need to log here anymore?

**Review by lmauromb — APPROVED** [review]

File: apps/meteor/app/livechat/server/methods/getAnalyticsOverviewData.ts (hunk 3 above)
Comment (KevLehman) [reply apps/meteor/app/livechat/server/methods/getAnalyticsOverviewData.ts:30]: Same here 👀

**Pull request merged**